Resources/Prototypes/_DEN/Entities/Mobs/Species/ovinia.yml

```diff
@@ -0,0 +1,136 @@
+        # its fine, but its still very stupid that it has to be done like this instead of allowing sprites to just directly insert a stencil clear.
+        # sprite refactor when
+        state: l_leg
+      - map: [ "enum.HumanoidVisualLayers.UndergarmentBottom" ] #Removed - Delta-V
+      - map: [ "enum.HumanoidVisualLayers.UndergarmentTop" ] #Removed - Delta-V
+      - map: [ "jumpsuit" ]
+      - map: [ "enum.HumanoidVisualLayers.LHand" ]
+      - map: [ "enum.HumanoidVisualLayers.RHand" ]
+      - map: [ "enum.HumanoidVisualLayers.LFoot" ]
+      - map: [ "enum.HumanoidVisualLayers.RFoot" ]
+      - map: [ "enum.HumanoidVisualLayers.Handcuffs" ]
+        color: "#ffffff"
+        sprite: Objects/Misc/handcuffs.rsi
+        state: body-overlay-2
+        visible: false
+      - map: [ "id" ]
+      - map: [ "gloves" ]
+      - map: [ "shoes" ]
+      - map: [ "ears" ]
+      - map: [ "outerClothing" ]
+      - map: [ "eyes" ]
+      - map: [ "belt" ]
+      - map: [ "neck" ]
+      - map: [ "back" ]
+      - map: [ "enum.HumanoidVisualLayers.FacialHair" ]
+      - map: [ "enum.HumanoidVisualLayers.HeadSide" ]
+      - map: [ "enum.HumanoidVisualLayers.NeckFluff" ]
+      - map: [ "enum.HumanoidVisualLayers.Hair" ]
+      - map: [ "enum.HumanoidVisualLayers.HeadTop" ]
+      - map: [ "enum.HumanoidVisualLayers.Tail" ]
+      - map: [ "mask" ]
+      - map: [ "head" ]
+      - map: [ "pocket1" ]
+      - map: [ "pocket2" ]
+  - type: HumanoidAppearance
+    species: Ovinia
+  - type: Fixtures
+    fixtures: # TODO: This needs a second fixture just for mob collisions.
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.35
+        density: 215 # sheep be heavy
+        restitution: 0.0
+        mask:
+        - MobMask
+        layer:
+        - MobLayer
+  - type: Body
+    prototype: Ovinia
+  - type: Damageable
+    damageModifierSet: Ovinia
+  - type: Speech
+    speechSounds: Alto
+    speechVerb: Ovinia
+    allowedEmotes: ["Baa"]
+  - type: Vocal
+    sounds:
+      Male: MaleOvinia
+      Female: FemaleOvinia
+      Unsexed: MaleOvinia
+  # type: LanguageKnowledge //Removed - Delta-V
+  #  speaks:
+  #  - TauCetiBasic
+  #  - Caprine
+  #  understands:
+  #  - TauCetiBasic
+  #  - Caprine
+  - type: ProtectedFromStepTriggers
+    #whitelist:
+    #  types:
+    #    - Shard
+    #    - Mousetrap
+    #    - SlipEntity //Removed - Delta-V
+  - type: Tag
+    tags:
+    - CanPilot
+    - FootstepSound
+    - DoorBumpOpener
+    - OviniaEmotes
+  - type: TypingIndicator
+    proto: ovinia
+  - type: Inventory
+    speciesId: ovinia
+    displacements:
+      jumpsuit:
+        sizeMaps:
+          32:
+            sprite: _DEN/Mobs/Species/Ovinia/displacement.rsi
+            state: jumpsuit
+      shoes:
+        sizeMaps:
+          32:
+            sprite: _DEN/Mobs/Species/Ovinia/displacement.rsi
+            state: shoes
+  - damage:
+      type:
+        firestacks: -0.2 #wool isnt flammable
```

Comment:
Just get rid of this `- damage` node. Its not valid in the slightest.

Every node in component must have `- type: [componentName]`. This is just `- damage` and its why its blowing up. Ill look how flammibility is implemented in the component on Den. It may be custom, so for now, just get rid if this.

Reply:
I dont think this does anything on Den. I find no logic when I search for references of `FireStackIncreaseMultiplier`. I see it in YML and the component but no where near else lol